utils/rxiv-download/README.md

```diff
@@ -0,0 +1,30 @@
+# Rxiv Download (0.1.0-dev)
+
+This plugin allows to download data from open access archives. Currently this plugin supports downloading data from  [arxiv](https://www.openarchives.org/). Later additional support for other archives will be added.
+
+## Building
+
+To build the Docker image for the download plugin, run
+`bash build-docker.sh`.
+
+## Run the Docker image
+
+To execute the built docker image for the download plugin, run
+`bash run-plugin.sh`.
+
+## Options
+
+This plugin takes 2 input arguments and
+1 output argument:
```

Comment:
2 input and 2 output, it seems?

Reply:
This plugin takes two input argument "rxiv" type Type of rxiv to be downloaded and "start" start date while one output directory "outDir" for saving downloaded ".xml" files